test.sh

```diff
@@ -21,8 +42,13 @@ test_files=(
     "$TEST_DIR/test_pytorch_helper.py"
 )
 
-if hash parallel 2>/dev/null; then
-    parallel -j 2 -t --line-buffer --keep-order $PYTHON {} -v ::: "${test_files[@]}"
+if [[ $PARALLEL == 1 ]]; then
+    if hash parallels 2>/dev/null; then
```

Comment:
parallels <-- is this the name? Not `parallel`?

Reply:
Nice catch, I was trying to go into poor man's version on my dev box but forgot to revert it :)